apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -24,6 +24,16 @@ export class EmailOutputRendererCommand extends RenderCommand {
   environmentId: string;
 }
 
+function isJsonString(str: string): boolean {
```

Comment:
here we could be more specific, and check for the maily json structure, although a simple parsing should validate if it is maily json or a string

Reply:
Perfect